### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by menyagah**
# In this section:
 - We added styling to our website
 - Added a functional navbar
 - Added the contact page and split the add book page with the display book page

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -2,6 +2,48 @@
...
 const author = document.querySelector('.author');
 const button = document.querySelector('.btn');
+const dateMonth = document.querySelector('.datetime');
+
+// DATE SECTION
+
+const section = document.createElement('div');
+section.classList.add('date-time');
+section.innerHTML = `<div>${Date()}<div>`;
+dateMonth.appendChild(section);
```
Comment (mimipeshy) [inline main.js]: - [x] This data should come from the [luxon library ](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time) don't use javascript's inbuilt date as you did. I can see you referenced the script in the html file but didn't use it in the js file. Kindly work on this

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @Team 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights

- Good job on the UI✔️ 
- Good GitHub workflow ✔️ 
- Not forming any linter errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Review by KabohaJeanMark — APPROVED** [review]
**Hi 👋🏾  @menyagah and @lily-coder,** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Thanks for the good work done in making the requested change.

I love the UI. It is bold and beautiful ✔️ 

[ ] OPTIONAL BUT HIGHLY RECOMMENDED FOR FUTURE PROJECTS

- Going forward, please use proper git workflow where you have the main branch, development branch and feature branch. 
- Then your main branch should initially be empty since it is for production. 
- You should raise a Pull Request from the feature branch to the development branch.

This [nice article](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) on git workflow on Atlassian explains this explicitly in a nice graphic way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by menyagah** [comment]
I really appreciate this very insightful and informative review @KabohaJeanMark. We realized the mistake in merging on the main branch a little too late but I intend NOT to repeat the same mistake again.
@mimipeshy Thank you so much for giving us thorough reviews that have enabled us to be keen on the project requirements and thereby enabling us to develop some good habits henceforth!

**Pull request merged**